src/Stripe.net/Entities/Accounts/Account.cs

```diff
@@ -14,7 +14,7 @@ public class Account : StripeEntity, IHasId, IHasMetadata, IHasObject, IPaymentS
         public string Object { get; set; }
 
         #region Expandable BusinessLogo
-        public string BusinessLogoFileId { get; set; }
+        public string BusinessLogoId { get; set; }
```

Comment:
Hmmmm Are we sure we want to get rid of the `File` part? This was a good way to infer something is the file id. The new name does not match the API name either.

Reply:
It's internally inconsistent with how other expandable fields are named and how attributes are named in general.

That said, I can add XML comments to indicate that this is a file ID.